test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/lang/CharacterTest.scala

```diff
@@ -383,6 +383,159 @@ class CharacterTest {
+    assertEquals('F', Character.toUpperCase('f'))
+    assertEquals('Z', Character.toUpperCase('z'))
+    // compat/ucd
+    assertEquals('Β', Character.toUpperCase('β'))
+    assertEquals('Μ', Character.toUpperCase('μ'))
+    // alt (what is this?)
+    assertEquals('Ā', Character.toUpperCase('Ā'))
+    assertEquals('Ā', Character.toUpperCase('ā'))
+    assertEquals('Ă', Character.toUpperCase('ă'))
+    assertEquals('Į', Character.toUpperCase('į'))
+    // high
+    assertEquals('Ａ', Character.toUpperCase('ａ'))
+    assertEquals('Ｚ', Character.toUpperCase('ｚ'))
+    assertEquals('［', Character.toUpperCase('［'))
+  }
+
+  @Test def toUpperCase_Int(): Unit = {
+    // Low
+    assertEquals(10, Character.toLowerCase(10)) // '\n' => '\n'
+    // Ascii
+    assertEquals(65, Character.toUpperCase(65)) // A => A
+    assertEquals(65, Character.toUpperCase(97)) // a => A
+    assertEquals(70, Character.toUpperCase(102)) // f => F
+    assertEquals(90, Character.toUpperCase(122)) // z => Z
+    // compat/ucd
+    assertEquals(0x0392, Character.toUpperCase(0x03D0)) // β => Β
+    assertEquals(0x039C, Character.toUpperCase(0x00B5)) // μ => Μ
+    // alt
+    assertEquals(256, Character.toUpperCase(256))
+    assertEquals(256, Character.toUpperCase(257))
+    assertEquals(258, Character.toUpperCase(259))
+    assertEquals(302, Character.toUpperCase(303))
+    // high
+    assertEquals(65313, Character.toUpperCase(65345))
+    assertEquals(65338, Character.toUpperCase(65370))
+    assertEquals(65339, Character.toUpperCase(65339))
+  }
+
+/*
+for(cp <- 0 until Character.MAX_CODE_POINT; if cp <= Character.MAX_VALUE) {
+  val strUpperCase: String = {
+    val s = new String(Array(cp), 0, 1)
+    s.toUpperCase()
+  }
+
+  if (strUpperCase.codePointCount(0, strUpperCase.length-1) > 1)
+    println(s"    assertEquals('${cp.toChar}', Character.toUpperCase('${cp.toChar}'))")
+}
+*/
+  @Test def toUpperCase_Char_MultiCodePoints(): Unit = {
+    assertEquals('ΐ', Character.toUpperCase('ΐ'))
+    assertEquals('ΰ', Character.toUpperCase('ΰ'))
+    assertEquals('ὒ', Character.toUpperCase('ὒ'))
+    assertEquals('ὔ', Character.toUpperCase('ὔ'))
+    assertEquals('ὖ', Character.toUpperCase('ὖ'))
+    assertEquals('ᾷ', Character.toUpperCase('ᾷ'))
+    assertEquals('ῇ', Character.toUpperCase('ῇ'))
+    assertEquals('ῒ', Character.toUpperCase('ῒ'))
+    assertEquals('ΐ', Character.toUpperCase('ΐ'))
+    assertEquals('ῗ', Character.toUpperCase('ῗ'))
+    assertEquals('ῢ', Character.toUpperCase('ῢ'))
+    assertEquals('ΰ', Character.toUpperCase('ΰ'))
+    assertEquals('ῧ', Character.toUpperCase('ῧ'))
+    assertEquals('ῷ', Character.toUpperCase('ῷ'))
+    assertEquals('ﬃ', Character.toUpperCase('ﬃ'))
+    assertEquals('ﬄ', Character.toUpperCase('ﬄ'))
+  }
+
+/*
+def format(codePoint: Int): String = "0x%04x".format(codePoint)
+
+for(cp <- 0 until Character.MAX_CODE_POINT) {
+  val strUpperCase: String = {
+    val s = new String(Array(cp), 0, 1)
+    s.toUpperCase()
+  }
```

Comment:
Consider making this shorter:
```suggestion
  val strUpperCase: String = new String(Array(cp), 0, 1).toUpperCase()
```